app/components/post_body/index.js

```diff
@@ -19,10 +21,14 @@ import {
 } from 'mattermost-redux/utils/post_utils';
 import {isAdmin as checkIsAdmin, isSystemAdmin as checkIsSystemAdmin} from 'mattermost-redux/utils/user_utils';
 
+import {hasEmojisOnly} from 'app/utils/emoji_utils';
+
 import PostBody from './post_body';
 
 const POST_TIMEOUT = 20000;
 
+const memoizeHasEmojisOnly = memoizeResult((message, customEmojis) => hasEmojisOnly(message, customEmojis));
```

Comment:
This is memoizing one result for all posts, so each post will run over the results from the other ones. You can fix that though by using a `makeMapStateToProps` and putting this inside of it so that each component gets its own copy of this

Reply:
Thanks @hmhealey, PR updated.